CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 2.35.0-rc0 / 2022-04-06
+
+* [CHANGE] TSDB: Delete *.tmp WAL files when Prometheus starts. #10317
+* [CHANGE] promtool: Add new flag `--lint`  for the commands `check rules` and `check config`. #10435
+* [ENHANCEMENT] Prometheus is built with Go 1.18. #10501
+* [ENHANCEMENT] Log gomaxprocs messages. #10506
```

Comment:
The actual feature is experimental support for gomaxprocs behind a feature flag - #10498

Reply:
ah I missed this one.
Thank you, Changelog is updated.